themes/startwords/assets/js/notes.js

```diff
@@ -0,0 +1,58 @@
+/* javascript for notes behavior */
+
+window.onload = function() {
+	// position contextual note if the page is loaded with one selected
+	// FIXME: not displaying on load - is this due to scrolling ?
+	positionContextualNote();
```

Comment:
you could try using the `document.addEventListener('DOMContentLoaded')` hook instead; that's guaranteed to fire before css takes effect and might have different results. we use that when setting up search pages in s&co to avoid tearing/visual movement of things.